client/cody-shared/src/sourcegraph-api/graphql/client.ts

```diff
@@ -150,6 +192,27 @@ export class SourcegraphGraphQLAPIClient {
         )
     }
 
+    public async isCodyEnabled(): Promise<{ enabled: boolean; version: string }> {
+        // Check site version. Cody does not work on versions older than 5.0
+        const siteVersion = await this.getSiteVersion()
+        if (isError(siteVersion)) {
+            return { enabled: false, version: '' }
+        }
+        const insiderBuild = siteVersion.length > 12 || siteVersion.includes('dev')
+        const versionBeforeCody = !insiderBuild && siteVersion < '5.0.0'
+        if (!insiderBuild && versionBeforeCody) {
```

Comment:
That condition is already contained in `versionBeforeCody`

```suggestion
        if (versionBeforeCody) {
```